includes/Discovery.php

```diff
@@ -60,12 +60,17 @@ public function init() {
 		add_action( 'web_stories_story_head', 'rsd_link' );
 		add_action( 'web_stories_story_head', 'wlwmanifest_link' );
 		add_action( 'web_stories_story_head', 'adjacent_posts_rel_link_wp_head', 10, 0 );
-		add_action( 'web_stories_story_head', 'noindex', 1 );
 		add_action( 'web_stories_story_head', 'wp_generator' );
 		add_action( 'web_stories_story_head', 'rel_canonical' );
 		add_action( 'web_stories_story_head', 'wp_shortlink_wp_head', 10, 0 );
 		add_action( 'web_stories_story_head', 'wp_site_icon', 99 );
 		add_action( 'web_stories_story_head', 'wp_oembed_add_discovery_links' );
+		// Add support for WP 5.7. See https://core.trac.wordpress.org/ticket/51511.
+		if ( function_exists( '\wp_robots' ) ) {
+			add_action( 'web_stories_story_head', 'wp_robots', 1 );
```

Comment:
Looks like PHPStan is not smart enough to not throw an error here, despite the `function_exists` check.

I suppose we would have to wait until `php-stubs/wordpress-stubs` is updated for 5.7 (which only happens upon release).

So we could do the following in the meantime:

1. Create a new file `tests/phpstan/stubs/wp-core.php`
2. Add this file to the `scanFiles` section in `phpstan.neon.dist`
3. Add the following to the file:  
  ```php
  /**
   * Displays the robots meta tag as necessary.
   *
   * @return void
   */
  function wp_robots() {
  }
  ```

Reply:
4. Remove this workaround once 5.7 is out. Opened #5998 to track this in general.

Reply:
I have a different workaround. https://github.com/google/web-stories-wp/pull/5997/commits/5dfdbf3f44dd594e55d983d2cfa0a83442833985

PHPstan is wrong here, so let's ignore it.